### PR TITLE
[RLlib] Fix two RL docs examples

### PR DESCRIPTION
**Pull request by ArturNiederfahrenhorst**
## Why are these changes needed?

We currently have two RLlib doc tests failing.
This PR fixes both.

### CODE REVIEW

File: rllib/core/models/catalog.py [hunk 1]
```diff
@@ -151,8 +151,8 @@ def build_my_head(self, framework: str):
 
         # With that, RLlib can build and use models from this catalog like this:
         catalog = MyCatalog(gym.spaces.Box(0, 1), gym.spaces.Box(0, 1), {})
-        my_head = catalog.build_my_head("torch")  # doctest: +SKIP
-        out = my_head(...)  # doctest: +SKIP
+        my_head = catalog.build_my_head("torch")
+        out = my_head(torch.Tensor([[1]]))
```
Comment (ArturNiederfahrenhorst) [inline rllib/core/models/catalog.py:155]: When I wrote them, I assumed that these `# doctest: +SKIP` would work on them. It does not.

File: rllib/algorithms/algorithm_config.py [hunk 2]
```diff
@@ -2593,35 +2593,25 @@ def get_multi_agent_setup(
...
+            from ray.rllib.algorithms.ppo import PPOConfig
+            config = (
+              PPOConfig()
+              .environment("CartPole-v1")
+              .framework("torch")
+              .multi_agent(policies={"pol1", "pol2"}, policies_to_train=["pol1"])
+            )
+            policy_dict, is_policy_to_train = config.get_multi_agent_setup(
+                env=gym.make("CartPole-v1"))
+            is_policy_to_train("pol1")
```
Comment (avnishn) [inline rllib/algorithms/algorithm_config.py:2608]: do you want to delete the output of these commands? otherwise lgtm

**Comment by pcmoritz** [comment]
Thanks for making the docs build green again ❤️ 

#fasterandgreener

**Review by gjoliver — APPROVED** [review]
finally. thanks a ton.

**Pull request merged**